dashboard/app/helpers/levels_helper.rb

```diff
@@ -574,4 +574,10 @@ def redirect_applab_under_13(level)
       return true
     end
   end
+
+  def can_view_solution?
+    if current_user && @level.try(:ideal_level_source_id) && @script_level && !@script.hide_solutions?
+      Ability.new(current_user).can? :view_level_solutions, @script
```

Comment:
Hmm why do we need to create an `Ability` here?  Does the `can?` helper only exist in controllers?